### PR TITLE
Add product-summary extension in header/minicart

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?
Add the product-summary extension in pages.json

#### What problem is this solving?
Adding an extension in header to  load the product-summary in the minicart

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
```
Comment (lucasecdb) [inline CHANGELOG.md]: remove :pick:

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 ### Added
 - Dreamstore with Design Tokens! :tada 
 - Extension point definition for not found results page.
+- Add `ProductSummary` to `header/minicart/product-summary` extension point.
+
```
Comment (lucasecdb) [inline CHANGELOG.md]: remove :pick:

File: CHANGELOG.md (hunk 1 above)
Comment (ArthurSampaio) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (ArthurSampaio) [reply CHANGELOG.md]: done

**Pull request merged**